packages/flutter_tools/lib/src/build_runner/devfs_web.dart

```diff
@@ -388,7 +389,8 @@ class WebDevFS implements DevFS {
       enableDwds,
       entrypoint,
     );
-    return Uri.parse('http://$hostname:$port');
+    _baseUri = Uri.parse('http://$hostname:$port');
```

Comment:
This might not be the correct information depending on what you need to connect to. For example, `hostname` might be something like a corp workstation hostname that needs to be resolved to an actual network address

Reply:
For our use case, the application and test will be on the same machine so this should be fine.